build/Stride.sln

```diff
@@ -374,6 +374,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Avalonia", "Avalonia", "{C4
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Plugins", "Plugins", "{BFF9AA54-F9DD-42BB-8163-6C03E7F51623}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Stride.Core.Assets.Editor.Avalonia", "..\sources\editor\Stride.Core.Assets.Editor.Avalonia\Stride.Core.Assets.Editor.Avalonia.csproj", "{67142D11-FA16-47E3-9894-6EE071F4F3B2}"
```

Comment:
I also added this project in my `xplat/property-grid` branch. If possible can you cherry-pick or copy this project creation so we avoid conflicts (esp. on the .sln).

I'll open a draft PR also for that branch so it's easier to find.